packages/block-library/src/block/edit.js

```diff
@@ -141,17 +196,15 @@ export default function ReusableBlockEdit( {
 
 	return (
 		<RecursionProvider uniqueId={ ref }>
-			<InspectorControls>
-				<PanelBody>
-					<TextControl
-						label={ __( 'Name' ) }
-						value={ title }
-						onChange={ setTitle }
-						__nextHasNoMarginBottom
-						__next40pxDefaultSize
-					/>
-				</PanelBody>
-			</InspectorControls>
+			{ userCanEdit && (
+				<BlockControls>
+					<ToolbarGroup>
+						<ToolbarButton href={ editUrl }>
```

Comment:
Not a blocker for this PR, but ideally we could also use client side routing here if the user is already inside the site editor.

Reply:
Let's wait and see what design says about the flows between editors and possibly tidy this up in a follow up.